rules-tests/TypeDeclaration/Rector/ClassMethod/StrictArrayParamDimFetchRector/Fixture/skip_array_append.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictArrayParamDimFetchRector\Fixture;
+
+final class SkipArrayAppend
+{
+    public function addSuccessMessage($success)
+    {
+        $this->success[] = $success;
```

Comment:
interessting.. the actual bug only happened because of the property-fetch involved had the same name as the parameter itself